src/drivers/drv_hrt.h

```diff
@@ -92,14 +92,22 @@ __EXPORT extern hrt_abstime ts_to_abstime(const struct timespec *ts);
  */
 __EXPORT extern void	abstime_to_ts(struct timespec *ts, hrt_abstime abstime);
 
+/**
+ * Compute the delta between a timestamp taken in the past
+ * and now.
+ *
+ * This function is not interrupt save.
+ */
+__EXPORT extern hrt_abstime hrt_elapsed_time(const volatile hrt_abstime *then);
```

Comment:
Can you change this to an inline method and remove the volatile?
```
static inline hrt_abstime hrt_elapsed_time(const hrt_abstime *then)
{
	return hrt_absolute_time() - *then;
}
```

Reply:
I didn't trust GCC and checked: it did not inline anything. Updated in https://github.com/PX4/Firmware/pull/11283